airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -35,7 +35,9 @@ Previously KubernetesPodOperator considered some settings from the Airflow confi
 Features
 ~~~~~~~~
 
-Previously, ``name`` was a required argument for KubernetesPodOperator (when also not supplying pod template or full pod spec). Now, if ``name`` is not supplied, ``task_id`` will be used.
+* Previously, ``name`` was a required argument for KubernetesPodOperator (when also not supplying pod template or full pod spec). Now, if ``name`` is not supplied, ``task_id`` will be used.
+* KubernetsPodOperator argument ``namespace`` is now optional.  If not supplied, we'll check the airflow conn,
```

Comment:
Should we document this priority?

Reply:
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-cncf-kubernetes/operators.rst